### PR TITLE
0.4 score

### PR DESCRIPTION
**Pull request by Andrewchistyakov**

### CODE REVIEW

File: dogs-v1.ipynb [hunk 1]
```diff
@@ -0,0 +1 @@
..."iopub.status.busy":"2025-07-20T00:06:51.762567Z","iopub.execute_input":"2025-07-20T00:06:51.762919Z","iopub.status.idle":"2025-07-20T00:06:51.768964Z","shell.execute_reply.started":"2025-07-20T00:06:51.762889Z","shell.execute_reply":"2025-07-20T00:06:51.768368Z"}},"outputs":[{"execution_count":9,"output_type":"execute_result","data":{"text/plain":"array([[7.5258185e-06, 5.0839276e-07, 9.7379029e-07, ..., 5.3913215e-07,\n        5.1961138e-07, 3.4672835e-06],\n       [3.6636200e-06, 1.6340374e-06, 3.0037386e-06, ..., 3.7134860e-06,\n        2.9057451e-06, 2.7767990e-06],\n       [1.3492114e-06, 4.8378839e-05, 1.0705379e-05, ..., 9.0500347e-05,\n        2.1239601e-04, 4.2485058e-06],\n       ...,\n       [1.2102249e-04, 3.0523763e-05, 5.4100563e-04, ..., 4.2828586e-05,\n        2.0455975e-05, 2.2910501e-05],\n       [3.0376992e-04, 2.0507237e-04, 7.6926015e-05, ..., 4.6254732e-05,\n        4.1808395e-05, 1.8616924e-04],\n       [4.5591364e-06, 4.3563519e-04, 5.5671630e-05, ..., 5.8613764e-04,\n        1.7253800e-05, 3.2371150e-07]], dtype=float32)"},"metadata":{}}],"execution_count":9},{"cell_type":"code","source":"","metadata":{"trusted":true},"outputs":[],"execution_count":null}]}
```
Comment (dpaleyev) [inline dogs-v1.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #4.    datagen = ImageDataGenerator(
</pre><p> </p><p>я не шарю за керас но чет слишком много аугментаций накручено. Я бы точно шифты и зум убрал - так как в шатаете и так много картинок кропнуто, а так только уменьшаешь количество данных.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Andrewchistyakov/summerml/pull/2/discussion/#comment-2233896133'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (dpaleyev) [inline dogs-v1.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #4.    model = keras.Sequential([
</pre><p>Вообще не стоит так делать, максимум что мы меняем в голове - количество выходных слоёв. Для этих нужд отлично справится библиотека timm. Там модели получше обучены </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Andrewchistyakov/summerml/pull/2/discussion/#comment-2233896134'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (dpaleyev) [inline dogs-v1.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #25.        optimizer=keras.optimizers.Adam(learning_rate=0.001),
</pre><p>Кажется высоковат Learning rate, я бы с ним поигрался</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Andrewchistyakov/summerml/pull/2/discussion/#comment-2233896135'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>
